boards/ESP32.md

```diff
@@ -212,7 +229,7 @@ be quite reliable.
 #### minicom / CuteCom
 
 There are other methods to connect to your ESP32, not just Espruino Web IDE.  Two
-very basic tools are [minicom](https://en.wikipedia.org/wiki/Minicom),
+very basic tools are [CoolTerm](http://freeware.the-meiers.org/),
```

Comment:
Please can you leave this as minicom, or at least add CoolTerm as another option? Minicom is probably the easiest solution for those running linux